packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-modal.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import {
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    UILoader,
+    PageOverlay,
+    Div100vhContainer,
+    HintBox,
+    Text,
+} from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import AdditionalKycInfoForm from './additional-kyc-info-form';
+
+type TAdditionalKycInfoFormWithHintBox = {
+    error: string;
+    setError: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const AdditionalKycInfoFormWithHintBox = ({ error, setError }: TAdditionalKycInfoFormWithHintBox) => {
+    return (
+        <React.Fragment>
+            {error && (
+                <HintBox
```

Comment:
@shahzaib-deriv Same here, Please use `<InlineMessage />` which is based on design system, Also if the design given is different with what you get from `<InlineMessage />` you need to ask the designers to fix the design and follow the design system 🙏🏻